infra/.#build.sh

```diff
@@ -0,0 +1 @@
+swnelson@swnelson-laptop.11282:1629218661
```

Comment:
What is this part for?

Reply:
Oops - that's a misplaced emacs temp file from doing `git add ./infra`; I shouldn't commit in a rush. Will remove.